build-scripts/gulp/translations.js

```diff
@@ -43,7 +43,6 @@ const TRANSLATION_FRAGMENTS = [
   "logbook",
   "mailbox",
   "profile",
-  "shopping-list",
```

Comment:
Should this be put back.

Reply:
Or maybe even better, we should auto-generate this based on the `src/panels` folders and then add the pages.

Reply:
There are no translations in the panel 🤷‍♂️